locales/mr/translation.json

```
@@ -0,0 +1,256 @@
+{
+    "$Language": {
+        "en": "इंग्रजी",
+        "es": "स्पॅनिश",
+        "it": "इटालियन",
+        "pt-BR": "पोर्तुगीज - ब्राझिल",
+        "mr":"मराठी"
+    },
+    "$Preferences": {
+        "title": "Time to Leave - प्राधान्ये",
+        "userPreferences": "वापरकर्ता प्राधान्ये",
+        "workingDays": "कामाचे दिवस",
+        "sun": "रवी",
+        "mon": "सोम",
+        "tue": "मंगळ",
+        "wed": "बुध",
+        "thu": "गुरू",
+        "fri": "शुक्र",
+        "sat": "शनि",
+        "hideNonWorkingDay": "नॉन-कामकाजाचे दिवस लपवा (महिना दृश्य)",
+        "hoursPerDay": "दिवसाचे तास",
+        "appBehavior": "अ‍ॅप वर्तन",
+        "notification": "सूचना",
+        "allowRecurringNotifications": "आवर्ती सूचनांना परवानगी द्या",
+        "minutesBetweenNotifications": "सूचना दरम्यान मिनिटे",
+        "startOnLogin": "लॉगिन वर प्रारंभ करा",
+        "countTodayOnTotals": "आज मोजा",
+        "overallBalanceStart": "एकूणच शिल्लक प्रारंभ तारीख",
+        "minimizeToTray": "ट्रेमध्ये मिनिमाइझ बटण कमीतकमी केले पाहिजे",
+        "closeMinimizeToTray": "ट्रेमध्ये क्लोज बटण कमीतकमी कमी करावे",
+        "themes": "थीम्स",
+        "systemDefault": "सिस्टम डीफॉल्ट",
+        "light": "प्रकाश",
+        "dark": "गडद",
+        "cadentStar": "कॅडेन्ट स्टार",
+        "view": "पहा",
+        "month": "महिना",
+        "day": "दिवस",
+        "numberOfEntries": "प्रवेशांची संख्या",
+        "fixed": "निश्चित",
+        "flexible": "परिवर्तनशील",
+        "language": "भाषा"
+    },
+    "$Menu": {
+        "about": "बद्दल",
+        "all-clear": "सर्व साफ!",
+        "all-files": "सर्व फायली",
+        "cancel": "रद्द करा",
+        "check-for-updates": "Check for updates",
+        "clear-database": "डेटाबेस साफ करा",
+        "confirm-clear-all-data": "आपली खात्री आहे की आपण सर्व डेटा साफ करू इच्छिता?",
+        "confirm-import-db": "आपली खात्री आहे की आपण डेटाबेस आयात करू इच्छिता? हे कोणतीही सद्य माहिती अधिलिखित करेल.",
+        "confirm-migrate-to-flexible": "आपल्याला खात्री आहे की आपण निश्चित कॅलेंडरमध्ये निश्चित कॅलेंडर डेटाबेस स्थानांतरित करू इच्छिता? \ N \ n विद्यमान लवचिक कॅलेंडर डेटाबेस साफ केला जाईल.",
```

I think the `\n` is incorrect here. Maybe it should be:
```suggestion
        "confirm-migrate-to-flexible": "आपल्याला खात्री आहे की आपण निश्चित कॅलेंडरमध्ये निश्चित कॅलेंडर डेटाबेस स्थानांतरित करू इच्छिता?\n\nविद्यमान लवचिक कॅलेंडर डेटाबेस साफ केला जाईल.",
```